mountain-climbing/src/main/java/com/example/mountainclimbing/service/ImageService.java

```diff
@@ -17,27 +20,32 @@ public ImageService(ImageRepository imageRepository) {
 		this.imageRepository = imageRepository;
 	}
 
-	public Image createImage(Image image, HttpServletRequest req, HttpServletResponse res) {
+	public ResponseEntity<Image> createImage(Image image, HttpServletRequest req, HttpServletResponse res) {
 		imageRepository.save(image);
-		return image;
+		return new ResponseEntity<Image>(image,HttpStatus.OK);
 	}
 	
 	public Optional<Image> readImage(int id, HttpServletRequest req, HttpServletResponse res) {
 		return imageRepository.findById(id);
 	}
 	
 	
-	public Image updateImage(int id,Image image, HttpServletRequest req, HttpServletResponse res) throws Exception {
+	public ResponseEntity<Image> updateImage(int id,Image image, HttpServletRequest req, HttpServletResponse res) {
```

Comment:
Add binding result and check if erros exists and return appropriate response entity and status code